OpenRA.Mods.Common/Traits/BotModules/BaseBuilderBotModule.cs

```diff
@@ -139,15 +136,36 @@ public class BaseBuilderBotModuleInfo : ConditionalTraitInfo
 		[Desc("Delay (in ticks) between reassigning rally points.")]
 		public readonly int AssignRallyPointsInterval = 100;
 
+		[Desc("Delay (in ticks) between finding a good resource point around conyard to harvest.")]
```

Comment:
```suggestion
		[Desc("Delay (in ticks) for finding a good resource to place a refinery next to.")]
```

Would this be better?